src/inference/include/ie/gna/gna_config.hpp

```diff
@@ -53,6 +53,13 @@ DECLARE_GNA_CONFIG_KEY(PRECISION);
  */
 DECLARE_GNA_CONFIG_KEY(FIRMWARE_MODEL_IMAGE);
 
+/**
+ * @brief Generation of GNA embedded device to export the model.
+ * @deprecated Key is deprecated and will be removed in 2022.2.
```

Comment:
Let's put it this way: "will be removed in a future release". I would not commit that each option will be removed in 2022.2.

Reply:
done